### PR TITLE
Obsolete Content-MD5

### LINKED ISSUE #93: Content-MD5
The deprecation of Content-MD5 in 7231 was a bit too subtle; it's only mentioned in a change note, and the registry wasn't updated.

### PR DESCRIPTION
**Pull request by mnot**
Fixes #93

### CODE REVIEW

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -9584,9 +9584,14 @@ Content-Encoding: gzip
    appropriate link.
 </t>
 <t>
-   After that is complete, please update the registry with the
+   After that is complete, please update the new registry with the
```
Comment (reschke) [inline draft-ietf-httpbis-semantics-latest.xml]: unrelated change..., would prefer it to appear in a separate commit :-)

File: draft-ietf-httpbis-semantics-latest.xml [hunk 2]
```diff
@@ -9584,9 +9584,14 @@ Content-Encoding: gzip
...
    header field names listed in the table of <xref target="field.names"/>.
 </t>
+<t>
+   Finally, please update the "Content-MD5" entry in the new registry to have
+   a status of 'obsoleted', and a reference of RFC7231, Appendix B, as this
+   instruction was omitted from that specification.
+</t>
 </section>
```
Comment (reschke) [inline draft-ietf-httpbis-semantics-latest.xml]: change

 RFC7231, Appendix B

to

\<xref target="RFC7231" x:fmt="of" x:sec="B"/>

**Comment by reschke** [comment]
I think this is ready.

**Comment by royfielding** [comment]
I think the registry is supposed to reference the obsolete spec where it was last defined, not the spec that mentions it briefly in a change note. In this case, it should be referring to RFC2616 Section 14.15.

**Comment by reschke** [comment]
@royfielding - I think the IANA registry should point to both - pull request updated.
